operators/multiclusterobservability/manifests/base/alertmanager/alertmanager-accessor-serviceaccount.yaml

```diff
@@ -4,3 +4,4 @@ metadata:
   name: observability-alertmanager-accessor
   labels:
     alertmanager: observability
+automountServiceAccountToken: false
```

Comment:
I think we'd probably want this to be part of the main PR instead? I assume this would cause some problem unless merged with the rest of https://github.com/stolostron/multicluster-observability-operator/pull/2197 .